clang/test/SemaCXX/ms_struct.cpp

```diff
@@ -39,6 +40,4 @@ struct C {
 
 static_assert(__builtin_offsetof(C, n) == 8,
               "long long field in ms_struct should be 8-byte aligned");
-#if !defined(TEST_FOR_ERROR) && !defined(TEST_FOR_WARNING)
-// expected-no-diagnostics
-#endif
+// expected-warning@-2 {{offset of on non-standard-layout type 'C'}}
```

Comment:
Missing EOL. Did you run `clang-format`?

Reply:
Yes. I ran `git clang-format HEAD~2`, but no files were modified. Should we apply clang-format to the tests as well? (I think removing EOL is not good. I'll fix it)

Reply:
We do not run clang-format on tests - in part because tests are sensible to line number changes.

Reply:
Ah I missed that it was in a test. We shouldn't run clang-format then. But the EOL should be fixed! Sorry for the drive-by review :)

Reply:
Nah, thanks a lot for spotting that, I didn't